stripe/src/main/java/com/stripe/android/UidParamsFactory.kt

```diff
@@ -0,0 +1,51 @@
+package com.stripe.android
+
+import android.content.Context
+import java.util.HashMap
+
+internal class UidParamsFactory constructor(
+    private val packageName: String,
+    private val uidSupplier: Supplier<StripeUid>
+) {
+
+    fun create(): Map<String, String> {
```

Comment:
should we name this createParams or something? Since we already have an object level method named `create`?

Reply:
I'll do it in followup 👍